ports/mimxrt/boards/common.ld

```diff
@@ -90,10 +92,9 @@ SECTIONS
     . = ALIGN(4);
   } > m_interrupts
 
-  __VECTOR_RAM = __Vectors;
-  __RAM_VECTOR_TABLE_SIZE_BYTES = 0x0;
+  __Vectors_RAM = itcm_start;
```

Comment:
Should this be `__Vectors_RAM = ORIGIN(m_isr)` instead?  I know it's the same thing, but `m_isr` seems more logical, and the source of truth here.

Reply:
That should work.